scripts/jams_to_lab.py

```diff
@@ -124,10 +124,10 @@ def convert_jams(jams_file, output_prefix, csv=False, comment_char='#', namespac
     # Make a counter object for each namespace type
     counter = collections.Counter()
 
-    annotations = set()
```

Comment:
If we're worried about uniqueness, it'd probably be safer to just dump all the annotations with `namespace` suffixes -- afaik, this is fine